docs/docs/05-reusable-components.md

```diff
@@ -223,7 +224,14 @@ Counter.defaultProps = { initialCount: 0 };
 
 ### No Autobinding
 
-Methods follow the same semantics as regular ES6 classes, meaning that they don't automatically bind `this` to the instance. You'll have to explicitly use `.bind(this)` or [arrow functions](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions) `=>`.
+Methods follow the same semantics as regular ES6 classes, meaning that they don't automatically bind `this` to the instance. You'll have to explicitly use `.bind(this)` or [arrow functions](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions) `=>`.  It is recommended that you bind your event handlers in the constructor over binding in the render function or using arrow functions so that a new function is not instantiated every render.
```

Comment:
Nits:

* double space before `It is`
* `every render` => `on every render` (IMO that reads better)